packages/pigeon/platform_tests/alternate_language_test_plugin/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -530,6 +531,7 @@
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
 				CURRENT_PROJECT_VERSION = 1;
+				GCC_TREAT_WARNINGS_AS_ERRORS = YES;
```

Comment:
This won't catch most of what I fixed here since it applies only to the code in RunnerTest; I forgot that when I first did it, but since we don't want warnings in our tests code itself either I decided to leave it. Real coverage of the output being warning-free will be covered by podspec-check once that's enabled.